Epsilon.Host.Frontend/src/components/SearchBox.vue

```diff
@@ -169,4 +276,41 @@ function displayValue(item: { name: string }): string {
 		}
 	}
 }
+
+.custom-box {
```

Comment:
Just know that the amount of precies px used will not make the application responsive